src/activity-functions-function-notation.xml

```diff
@@ -753,19 +807,79 @@
+                        <static>20</static>
+                    </var>
+                    <var name="$M">
+                        <static>M</static>
+                    </var>
+                    <var name="$Mexp">
+                        <static>M^2-M</static>
+                    </var>
+                    <var name="$Q">
+                        <static>Q</static>
+                    </var>
+                    <var name="$Qexp">
+                        <static>Q^2-Q</static>
+                    </var>
+                    <var name="$P">
+                        <static>x-12</static>
+                    </var>
+                    <var name="$expression">
+                        <static>(x-12)^2-(x-12)</static>
+                    </var>
+                    <pg-code>
+                        $five = Real(5);
+                        $out = Real(20);
+                        Context()->variables->are(M=>'Real', Q=>'Real', x=>'Real');
+                        $M=Compute("M");
+                        $Q=Compute("Q");
+                        $P=Compute("x-12");
+                        $Mexp=Compute("M**2-M");
+                        $Qexp=Compute("Q**2-Q");
+                        $expression=Compute("(x-12)**2-(x-12)");
+                    </pg-code>
+                </setup>
+
+                <statement>
+                    <p>Use the function <m>f(x) = x^2 - x</m> to evaluate <m>f(5)</m>.</p>
+                    <p>Step 1: Replace all the x's in the formula with parentheses to create blank spaces.</p>
+                    <me>f(blank) = (blank)^2 - (blank)</me>
+                    <p>Step 2: Identify the intput. In this case the given input is <var name="$five" width="2" /></p>
+                    <p>Step 3: Place the input into all the blank spaces</p>
+                    <p><m>f(5) = (</m> <var name="$five" width="2" /><m>)^2 - (</m> <var name="$five" width="2" /><m>)</m></p>
+                    <p>Step 4: Evaluate the function. <m>f(5) = </m><var name="$out" width="2" /></p>
+                    <p>Using the same method, evaluate each of the following expressions (do not simplify):</p>
+                    <p><m>f(M) = (</m> <var name="$M" width="2" /><m>)^2 - (</m> <var name="$M" width="2" /><m>)</m></p>
+                    <p><m>f(M) = </m> <var name="$Mexp" width="5" /></p>
+                    <p><m>f(Q) = (</m> <var name="$Q" width="2" /><m>)^2 - (</m> <var name="$Q" width="2" /><m>)</m></p>
+                    <p><m>f(Q) = </m> <var name="$Qexp" width="5" /></p>
+                    <p><m>f(x-12) = (</m> <var name="$P" width="2" /><m>)^2 - (</m> <var name="$P" width="2" /><m>)</m></p>
+                    <p><m>f(x-12) = </m> <var name="$expression" width="5" /></p>
+                </statement>
+                <solution>
+                    <p>In Math the variable <m>M</m> is completely different thant <m>m</m>. Be careful to distinguish between lower case and capital variables!</p>
```

Comment:
This sentence doesn't seem right to be inside a `<solution>`. It's information that should come before an answer blank is encountered (for accessibility reasons).